src/node_crypto.cc

```diff
@@ -586,24 +595,26 @@ int SSL_CTX_use_certificate_chain(SSL_CTX* ctx,
       // by SSL_CTX_use_certificate).
 
       // Find issuer
-      if (*issuer != nullptr || X509_check_issued(ca, x) != X509_V_OK)
+      if (!*issuer || X509_check_issued(ca, x.get()) != X509_V_OK)
```

Comment:
I might be missing something here, but doesn't this negate the check?

Reply:
@tniessen Yes, good catch. I don’t think we have tests for this at the moment. :/